tests/DependencyInjection/SonataHelpersExtensionTest.php

```diff
@@ -84,19 +89,33 @@ public function testPrepend(): void
             ->willReturn([[]]);
         $container->expects($this->at(1))->method('getExtensionConfig')
             ->with('sonata_media')
-            ->willReturn([['providers' => [
+            ->willReturn([$sonataMediaConfig]);
+        $container->expects($this->once())->method('prependExtensionConfig')
+            ->with('sonata_helpers', $expected);
+
+        $this->extension->prepend($container);
+    }
+
+    /**
+     * @return \Generator
+     */
+    public function prependDataProvider(): \Generator {
+        yield [
```

Comment:
all yields items should be named (raises deprecation with phpunit8 and exception with phpunit9)